x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_size_property.js

```diff
@@ -166,6 +163,14 @@ export class DynamicSizeProperty extends DynamicStyleProperty {
     );
   }
 
+  renderLegendDetailRow() {
+    const header = this.renderRangeLegendHeader();
```

Comment:
Keep these details as part of `OrdinalLegend`. So `OrdinalLegend` calls style. renderRangeLegendHeader and returns null if nothing is returned